MiSeq_runs_basespace.sh

```diff
@@ -75,18 +75,6 @@ do
         wait
         
         echo "Download complete! Creating trace file confirming download!"
-        touch /Volumes/IDGenomics_NAS/WGS_Serotyping/$run_name/download_complete.txt
-        
-        icav2 projects enter Testing \
-          2>> $icav2_log.err | tee -a $ica2_log.log
-        echo y | icav2 projectdata upload /Volumes/IDGenomics_NAS/WGS_Serotyping/$run_name/Sequencing_reads/Raw /$run_name/ \
-          2>> $icav2_log.err | tee -a $ica2_log.log
-        
-        wait
-        echo "Uploaded to ICA!"
-        
-        echo "Upload to ICA complete! Renaming trace file to upload_complete!"
-        mv /Volumes/IDGenomics_NAS/WGS_Serotyping/$run_name/download_complete.txt /Volumes/IDGenomics_NAS/WGS_Serotyping/$run_name/upload_complete.txt
         
         echo "$(date) : Now starting Grandeur" | tee -a $grandeur_log.log
```

Comment:
Your python scripts make MiSeq_runs_basespace.sh unnecessary, correct? I think that means we can delete it.